crates/nu-command/tests/commands/url/join.rs

```diff
@@ -366,3 +366,21 @@ fn url_join_with_fragment_and_params() {
         "http://usr:pwd@localhost:1234?par_1=aaa&par_2=bbb#frag"
     );
 }
+
+#[test]
+fn url_join_with_empty_params() {
+    let actual = nu!(
+        cwd: ".", pipeline(
+            r#"
+            {
+                "scheme": "https",
+                "host": "localhost",
+                "path": "/foo",
+            } | url join
```

Comment:
actually this was already equal to `https://localhost/foo` on the current `main` branch, so i think this is not the thing to test :thinking: 

however,
```bash
{
    scheme: https,
    host: example.com,
    path: /foo,
    params: {}
} | url join
```
is equal, from `cargo run`, to
- `https://example.com/foo?` on 82e6873702f2e3588e887915e2c5acdb54194bc0, i.e. latest `main` right now
- `https://example.com/foo` on the tip of this PR branch

so it looks to me this should be the test :yum: 

> **Note**
> the record was generated from `https://example.com/foo | url parse` :+1:

Reply:
Thanks for catching this. Fixed the unit test.